Sparrow.ps1

```diff
@@ -187,37 +187,37 @@ Function Get-UALData {
     #By default, it will show up as Consent_Operations_Export.csv       
     Export-UALData -ExportDir $ExportDir -UALInput $ConsentData -CsvName "Consent_Operations_Export" -WorkloadType "AAD"
 
-  #Searches for SAML token usage anomaly (UserAuthenticationValue of 16457) in the Unified Audit
-  $federatedDomains = Get-MsolDomain | Where-Object {$_.Authentication -eq "Federated"}
-  # Get only root domains so we can get SupportMFA status
-  $rootDomains = $federatedDomains | Where-Object {$_.RootDomain -eq $null}
-  # Get root domains that don't support MFA, hence Federated MFA is not expected. Note: federated MFA is still possible when SupportsMFA is false, however less likely. Check your STS configuration.
-  $rootDomainsSupportMFAFalse = @()
-  
-  foreach ($rootDomain in $rootDomains)
-  {
-    $fedProps = Get-MsolDomainFederationSettings -DomainName $rootDomain.Name 
-    If ($fedProps.SupportsMfa -eq $False) {
-        $rootDomainsSupportMFAFalse += $rootDomain.Name
+    #Searches for SAML token usage anomaly (UserAuthenticationValue of 16457) in the Unified Audit
+    $federatedDomains = Get-MsolDomain | Where-Object {$_.Authentication -eq "Federated"}
+    # Get only root domains so we can get SupportMFA status
+    $rootDomains = $federatedDomains | Where-Object {$_.RootDomain -eq $null}
+    # Get root domains that don't support MFA, hence Federated MFA is not expected. Note: federated MFA is still possible when SupportsMFA is false, however less likely. Check your STS configuration.
+    $rootDomainsSupportMFAFalse = @()
+    
+    foreach ($rootDomain in $rootDomains)
+    {
+        $fedProps = Get-MsolDomainFederationSettings -DomainName $rootDomain.Name 
+        If ($fedProps.SupportsMfa -ne $True) {
```

Comment:
I changed this line to -ne $True as if SupportsMfa is $null, it technically should be treated as $false (since it seems that the value was not set).